runhouse/main.py

```diff
@@ -217,35 +217,34 @@ def _print_cluster_config(cluster_config: Dict):
             )
 
 
-def _print_envs_info(
-    env_resource_mapping: Dict, env_servlet_processes: Dict, current_cluster: Cluster
-):
+def _print_envs_info(env_servlet_processes: Dict, current_cluster: Cluster):
```

Comment:
can we be more descriptive with the dict type here? it's a little unclear to me what its supposed to be